app/src/main/java/com/framgia/soundclound/screen/detailgenre/GenreDetailViewModel.java

```diff
@@ -0,0 +1,87 @@
+package com.framgia.soundclound.screen.detailgenre;
+
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports